tenets/codelingo/lightning-network-daemon/rpc-responses-display-uniformly/codelingo.yaml

```diff
@@ -0,0 +1,106 @@
+funcs:
+  - name: protoNameNotSnakeCase
+    type: asserter
+    body: |
+      function (a) {
+        function notSnakeCase(a) {
+          return a.indexOf(" ") || a.match(/[A-Z]/)
+        }
+        var m = a.match(/^protobuf:"[^"]*name=([^,"]+).+ json:"([^,"]+)/)
+        return m && notSnakeCase(m[1]) && notSnakeCase(m[2])
+      }
+  - name: protoNameNotEqualsJsonName
+    type: asserter
+    body: |
+      function (a) {
+        var m = a.match(/^protobuf:"[^"]*name=([^,"]+).+ json:"([^,"]+)/)
+        return m && m[1] != m[2]
+      }
+  - name: identNameNotMatchesProtoName
+    type: asserter
+    body: |
+      function (ident, a) {
+        function singleCapitalize(str) {
+          function capitalizeFirstLetter(string) {
+              return string.charAt(0).toUpperCase() + string.slice(1);
+          }
+          return capitalizeFirstLetter(str.replace(/_+/g, '').replace(/\s+/g, ''));
+        }
+        function camelize(str) {
+          function capitalizeFirstLetter(string) {
+              return string.charAt(0).toUpperCase() + string.slice(1);
+          }
+          return capitalizeFirstLetter(str.replace(/_+/g, ' ').replace(/(?:^\w|[A-Z]|\b\w)/g, function(letter, index) {
+            return index == 0 ? letter.toLowerCase() : letter.toUpperCase();
+          }).replace(/\s+/g, ''));
+        }
+        var m = a.match(/^`protobuf:"[^"]*name=([^,"]+)/)
+        return m && ! ( ident == camelize(m[1]) || ident == singleCapitalize(m[1]) )
+      }
+tenets:
+  - name: rpc-responses-display-uniformly-fields-equal
+    flows:
+      codelingo/docs:
+        title: RPC responses display uniformly with fields equal
+        body: |
+      codelingo/review:
+        comment: protobuf name and json_name are not equal here. Please change their values to match in the corresponding .proto file.
+    query: |
+      import codelingo/ast/go
+      
+      go.file(depth = any):
```

Comment:
remove go.file

Reply:
Unfortunately, are not able to remove as-is, but will try after squashing the tenets..
https://trello.com/c/sLnkYGsp/455-gofile-sometimes-not-implicit